app/cart/controller.js

```diff
@@ -2,12 +2,16 @@ import { alias } from '@ember/object/computed';
 import Controller from '@ember/controller';
 
 export default Controller.extend({
-  // cartItems: Ember.computed.map('model', function(item) {
-  //   return {
-  //     id:
-  //     name: item.get('product.name'),
-  //     quantity: item.get('quantity')
-  //   };
-  // })
-  cartItems: alias('model')
+  totalPrice: Ember.computed('cartItemCount', function() {
+    const cartItemCount = this.get('cartItemCount') * 25
+    return `$${cartItemCount}.00`
```

Comment:
```
;
```

Reply:
what's `25`?

Reply:
25 is the cost. we have `lowestPrice` on a product model, but we need to implement it in the CMS

Reply:
it's just a placeholder for now

Reply:
derp, cool